src/main/java/com/fundynamic/d2tm/game/map/MapEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.fundynamic.d2tm.game.map;
+
+import com.fundynamic.d2tm.game.behaviors.Selectable;
+import com.fundynamic.d2tm.math.Vector2D;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.SpriteSheet;
+
+/**
+ * An entity bound to the map
+ */
+public abstract class MapEntity {
```

Comment:
I feel the `Map` prefix is used inconsistently. We have it here, but it's `Unit` and not `MapUnit` or `MapStructure`. Maybe rename it to `Entity` or are we expecting other kinds of entities?